apps/web/src/pages/SupportPage/components/SupportForm/SupportForm.tsx

```diff
@@ -309,13 +276,52 @@ const SupportFormApi = () => {
     });
     if (response.ok) {
       logger.info("Ticket successfully submitted");
+      toast.success(
+        intl.formatMessage({
+          defaultMessage: "Ticket created successfully!",
+          id: "jHuiRm",
+          description: "Support form toast message success",
+        }),
+      );
       return;
     }
 
-    logger.error(
-      `Failed to submit ticket: ${response.status} - ${response.statusText}`,
+    // we didn't get an OK so let's take a closer look at the response
+    const responseBody = await response.json();
+    const errorCode = `${response.status} - ${response.statusText}`;
+    logger.error(`Failed to submit ticket: ${JSON.stringify(responseBody)}`);
+
+    // default error message if we don't recognize the error
+    let errorMessage = defineMessage({
+      defaultMessage:
+        "Sorry, something went wrong. Please email <anchorTag>{emailAddress}</anchorTag> and mention this error code: {errorCode}.",
+      id: "rNVDaA",
+      description: "Support form toast message error",
+    });
```

Comment:
I think we're moving to defining messages like this outside the component (just above usually) so they aren't getting redeclared.

Reply:
Good idea, thanks!
[pull messages out of component](https://github.com/GCTC-NTGC/gc-digital-talent/pull/9864/commits/d136f73078ddec9942ca324e46b6a7afa55ed501)